src/transform/src/notice/index_already_exists.rs

```diff
@@ -29,6 +29,8 @@ pub struct IndexAlreadyExists {
     pub index_key: Vec<MirScalarExpr>,
     /// The id of the object that the index is on.
     pub index_on_id: GlobalId,
+    /// The id the index that duplicates existing indexes.
+    pub exported_index_id: GlobalId,
```

Comment:
Naming nit: Maybe this would be clearer with `old_index_id`/`new_index_id` or `existing_index_id`/`new_index_id`, instead of `index_id`/`exported_index_id`?

Reply:
I agree, but that's the names that are used elsewhere, so I didn't want to touch it!